pkg/verifier/aws/entry_point.go

```diff
@@ -44,11 +48,59 @@ func (a *AwsVerifier) ValidateEgress(vei verifier.ValidateEgressInput) *output.O
 		configPath = fmt.Sprintf(CONFIG_PATH_FSTRING, helpers.PlatformAWS)
 	}
 
+	var debugPubKey []byte
+	// Check if Import-keypair flag has been passed
+	if vei.ImportKeyPair != "" {
+		//Read the pubkey file content into a variable
+		PubKey, err := os.ReadFile(vei.ImportKeyPair)
+		debugPubKey = PubKey
+		if err != nil {
+			return a.Output.AddError(err)
+		}
+
+		//Import Keypair into aws keypairs to be attached later to the created debug instance
+		_, err = a.AwsClient.ImportKeyPair(vei.Ctx, &ec2.ImportKeyPairInput{
```

Comment:
for consecutive runs against the same AWS account, the second and afterward executions fail due to the key is already imported 
We can check if the key is already exists, and clean-it-up before re-importing a new one with the same

```go
Using region: us-east-1
Summary:
printing out failures:
printing out exceptions preventing the verifier from running the specific test:
printing out errors faced during the execution:
 - error performing ec2:ImportKeyPair: The keypair already exists
 ```

Reply:
ah this is good catch Boran. I forgot to test this case. Will fix right now before merging

Reply:
As discussed in https://redhat-internal.slack.com/archives/CMD1L32M9/p1683299251940989, we might prefer having the error message instead of cycling the old key